packages/esm-patient-orders-app/src/index.ts

```diff
@@ -25,3 +27,13 @@ registerWorkspace({
 });
 
 export const orderBasketActionMenu = getSyncLifecycle(orderBasketActionMenuComponent, options);
+
+export const ordersDashboardLink = getSyncLifecycle(
+  createDashboardLink({
```

Comment:
```suggestion
	// t('Orders', 'Orders')
  createDashboardLink({
```

For support of translating the orders text